charts/seed-monitoring/charts/prometheus/values.yaml

```diff
@@ -189,6 +189,16 @@ allowedMetrics:
   vpn:
   - probe_http_status_code
   - probe_success
+  elasticsearch:
+  - elasticsearch_cluster_health_json_parse_failures
+  - elasticsearch_cluster_health_total_scrapes
+  - elasticsearch_cluster_health_up
```

Comment:
I checked the *_up metrics in my cluster and they all return 0. Is this expected?

Reply:
Yes, it is expected. 0 means that the last scrape of the elasticsearch has been successful

# HELP elasticsearch_index_stats_up Was the last scrape of the ElasticSearch index endpoint successful.

Reply:
Normally 1 means that the scrape or action was successful. Why is it 0 here?

Reply:
Hmm, then i am not sure about these 3 metrics.  All the documentation that i found for these metrics is the i posted above...

Reply:
```
err="failed to get index stats from localhost://:/_all/_stats: Get localhost:9200: unsupported protocol scheme \"localhost\""
```

Prefixing the url with `http://` should fix it.

Reply:
Yes, Thank you @ialidzhikov . Now it should be okay